QualificationChecker/Models/Organization.cs

```diff
@@ -10,11 +8,20 @@ public class Organization
 
         [Required(ErrorMessage = "Age requirement is required.")]
         [Range(14, 120, ErrorMessage = "Age requirement must be between 14 and 70.")]
-        public int AgeRequirement { get; set; }
+        public int MinAgeRequirement { get; set; }
 
         public List<OrgQuestion>? OrgQuestions { get; set; }
     }
 
+    public static class OrganizationExtensions
```

Comment:
Yay, I love extension methods!  Remember, a file should only have one class in it